src/FluxTraining.jl

```diff
@@ -103,7 +103,6 @@ export AbstractCallback,
     fit!,
     epoch!,
     step!,
```

Comment:
```suggestion
    step!,
    onecycle,
```
Let's make this less breaking